src/main/java/frc/robot/Config.java

```diff
@@ -400,7 +400,7 @@ public static enum TeleopSpeeds {
     public static final IdleMode driveNeutralMode = IdleMode.kBrake;
 
     /* Motor Inverts */
-    public static final boolean driveInvert = false;
+    public static final Boolean[] driveInvert = {true, false, false, false};
```

Comment:
Please add an inversion field in SwerveModuleConstants.
Then set each module in class Mod0 - Mod3